components/raftstore/src/coprocessor/split_check/keys.rs

```diff
@@ -195,18 +204,47 @@ pub fn get_region_approximate_keys_cf(
     db: &impl KvEngine,
     cfname: &str,
     region: &Region,
+    large_threshold: u64,
 ) -> Result<u64> {
     let start_key = keys::enc_start_key(region);
     let end_key = keys::enc_end_key(region);
     let range = Range::new(&start_key, &end_key);
-    let (mut keys, _) = box_try!(db.get_approximate_memtable_stats_cf(cfname, &range));
+    let mut total_keys = 0;
+    let (mem_keys, _) = box_try!(db.get_approximate_memtable_stats_cf(cfname, &range));
+    total_keys += mem_keys;
 
     let collection = box_try!(db.get_range_properties_cf(cfname, &start_key, &end_key));
     for (_, v) in collection.iter() {
         let props = box_try!(RangeProperties::decode(&v.user_collected_properties()));
-        keys += props.get_approximate_keys_in_range(&start_key, &end_key);
+        total_keys += props.get_approximate_keys_in_range(&start_key, &end_key);
     }
-    Ok(keys)
+
+    if total_keys > large_threshold && large_threshold != 0 {
+        let ssts = collection
+            .iter()
+            .map(|(k, v)| {
+                let props = RangeProperties::decode(&v.user_collected_properties()).unwrap();
+                let key = props.get_approximate_keys_in_range(&start_key, &end_key);
+                format!(
+                    "({} {})",
```

Comment:
Maybe print sst count is enough

Reply:
I think the log will be `ssts [1.sst 100, 2.sst 100, 3.sst 150 ...]`, I guess users cannot get what is 100 and 150 mean.